app/Repositories/Interfaces/WebhookRepositoryInterface.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace App\Repositories\Interfaces;
+
+interface WebhookRepositoryInterface
+{
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Interfaces/WebhookRepositoryInterface.php</br>**Line**: 8</br>Expected 1 newline at end of file; 0 found